.github/workflows/pyopenms-wheels.yml

```diff
@@ -300,7 +300,7 @@ jobs:
         ## Update the package lists for Brew
         brew update
         ## Needed for Qt. Install before to overwrite the default softlinks on the GH runners
-        brew install python3 --force --overwrite
+        brew install python@3.13 --force --overwrite
```

Comment:
3.13 correct? just saw 3.12 above without further looking into it

Reply:
err, correct, it should be 3.12.